static/app/utils/integrationUtil.tsx

```diff
@@ -150,6 +151,12 @@ export function isDocIntegration(
   return integration.hasOwnProperty('isDraft');
 }
 
+export function isExternalActorMapping(
+  mapping: ExternalActorMappingOrSuggestion
+): mapping is ExternalActorMapping {
```

Comment:
What is this difference between the `is` keyword and `as`?

Reply:
@mgaeta Since `ExternalActorMappingOrSuggestion` could be `ExternalActorMapping` or `ExternalActorSuggestion`, this function is meant to distinguish which type of mapping was provided to it.

e.g.
```javascript
const {mapping} = this.props // TS says 'mapping' is ExternalActorMappingOrSuggestion

if (isExternalActorMapping(mapping)) {
   return mapping // TS says 'mapping' is ExternalActorMapping
}  else {
   return mapping // TS says 'mapping' is still ExternalActorMappingOrSuggestion
}

```

The word as is usually to get more specific from generic types without actually checking it. For example, our API client on the frontend returns data as an `any` type for the most part. Since I know the shape of the return is `ExternalActorMapping`, I can just add that to the param so that typescript doesn't throw any errors when compiling:

e.g.
```typescript
...
onResults={(data: any) => {
  // TS says 'mapping' is ExternalActorMapping
  const mapping = data as ExternalActorMapping 
}
...
onChange={(data: string) => {
  // TS throws an error, shape of ExternalActorMapping and string are not compatible.
  const mapping = data as ExternalActorMapping 
}
...
```